common/src/main/java/com/redhat/cloud/notifications/models/dto/v1/endpoint/properties/WebhookPropertiesDTO.java

```diff
@@ -0,0 +1,85 @@
+package com.redhat.cloud.notifications.models.dto.v1.endpoint.properties;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.cloud.notifications.models.HttpType;
+import com.redhat.cloud.notifications.models.dto.v1.endpoint.properties.secrets.BasicAuthenticationDTO;
+import com.redhat.cloud.notifications.models.validation.ValidNonPrivateUrl;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+public final class WebhookPropertiesDTO extends EndpointPropertiesDTO {
+    @NotNull
+    @JsonProperty(value = "disable_ssl_verification", required = true)
+    private Boolean disableSslVerification = Boolean.FALSE;
+
+    @NotNull
+    @JsonProperty(required = true)
+    private HttpType method;
+
+    @NotNull
+    @JsonProperty(required = true)
+    @ValidNonPrivateUrl
+    private String url;
+
+    @JsonProperty("basic_authentication")
```

Comment:
That kind of annotations shouldn't be needed if `@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)` works on the parent class.